mainpage.dox

```diff
@@ -574,6 +584,123 @@ janus.attach(
  *
  */
 
+/*!\page js-dependencies Working with custom \c janus.js dependencies
```

Comment:
Can you remove the `\c` from the section title? Just a nit as it simply renders poorly when compiled by Doxygen.